src/services/photo/index.js

```diff
@@ -1,13 +1,13 @@
 const errors = require('feathers-errors');
-const GCSUploader = require('../../utils/gcs-uploader');
+const Promise = require('bluebird');
```

Comment:
I use this style in React projects (airbnb style enforces this rule but I'm not sure why they don't enforce this rule in airbnb-base).

Alternative style is to follow https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md. But, I think it might be too strict.